pixi.toml

```
@@ -1,25 +1,59 @@
-[project]
+[workspace]
 name = "aligator"
 version = "0.16.0"
 channels = ["conda-forge"]
 description = "A primal-dual augmented Lagrangian-type solver for trajectory optimization"
 platforms = ["linux-64", "osx-64", "osx-arm64"]
 license = "BSD-2-Clause"
 license-file = "LICENSE"
+readme = "README.md"
+repository = "https://github.com/Simple-Robotics/aligator"
+preview = ["pixi-build"]
 
-[build-dependencies]
+[package]
+name = { workspace = true }
+version = { workspace = true }
+
+[package.build]
+backend = { name = "pixi-build-cmake", version = "*" }
+
+[package.build.config]
+extra-args = [
+  "-DBUILD_TESTING=OFF",
+  "-DGENERATE_PYTHON_STUBS=ON",
+]
+
+[package.build-dependencies]
+cmake = ">=3.22"
 ccache = ">=4.9.1"
-cmake = ">=3.10"
 cxx-compiler = ">=1.7.0"
-ninja = ">=1.11"
 pkg-config = ">=0.29.2"
 
+[package.host-dependencies]
+libboost-devel = ">=1.86.0,<2"
+libboost-python-devel = ">=1.86.0,<2"
+eigen = ">=3.4.0,<4"
+python = ">=3.9.22"
+eigenpy = "*"
+fmt = ">=11.1.4"
+benchmark = ">=1.9.4,<2"
+pytest = ">=8.3.5,<9"
+matplotlib-base = ">=3.9.4,<4"
+typed-argument-parser = ">=1.10.1,<2"
+meshcat-python = ">=0.3.2,<0.4"
+catch2 = ">=3.8.0,<4"
```

```suggestion
typed-argument-parser = ">=1.10.1,<2"
```
Since testing is disabled, are these dependencies mandatory ?